app/browser/webtorrent.js

```diff
@@ -0,0 +1,102 @@
+const electron = require('electron')
+const ipc = electron.ipcMain
+const messages = require('../../js/constants/messages')
+const WebTorrent = require('webtorrent')
+
+var DEBUG_IPC = false // Set to see communication between WebTorrent and torrent viewer tabs
+var ANNOUNCE = [
+  'wss://tracker.btorrent.xyz',
+  'wss://tracker.openwebtorrent.com',
+  'wss://tracker.fastcast.nz'
+]
+
+var client = null
+var channels = {}
+
+ipc.on(messages.TORRENT_MESSAGE, function (e, msg) {
+  if (DEBUG_IPC) console.log('Received IPC: ' + JSON.stringify(msg))
+  channels[msg.channelID] = e.sender
+  handleMessage(msg)
+})
+
+function handleMessage (msg) {
+  switch (msg.type) {
+    case 'add':
+      return handleAdd(msg)
+    default:
+      // Sanity check. Is there a better way to do error logging in the browser process?
+      console.error('Ignoring unknown action ' + msg.type + ', channel ' + msg.channelID)
+  }
+}
+
+function handleAdd (msg) {
+  var torrent = lazyClient().add(msg.torrentID, {
```

Comment:
Nit: Throughout the webtorrent library at least, we go with `torrentId` instead of `torrentID`, `magnetUrl` instead of `magnetURL`, etc., etc.